src/main/webapp/WEB-INF/themes/desktop/xform/article-transform.xsl

```diff
@@ -161,6 +161,8 @@
 
   <!-- Ambra-specific template (creates article metadata) -->
   <xsl:template name="make-article-meta">
+    <xsl:variable name="isPreregisteredResearchReport"
+                  select="/article/front/article-meta/article-categories/subj-group[@subj-group-type='heading']/subject[text()='Preregistered Research Article']"/>
```

Comment:
I'm curious why you named this var `isPreregisteredResearchReport` instead of `isPreregisteredResearchArticle`

Reply:
lol no

Reply:
brain fart

Reply:
force pushed a fix